src/components/Images.vue

```diff
@@ -333,6 +336,9 @@ export default {
     imageTag(tag) {
       return tag === '<none>' ? 'latest' : `${ tag.trim() }`;
     },
+    isNotNoneImage(row) {
+      return row.imageName && row.imageName !== '' && row.imageName !== '<none>';
```

Comment:
I was contemplating if testing only for `imageName` would be enough but it sounds like this is the preferred approach after reading through the issue.